### ISSUE TITLE
Add 8 Storage Places as functionality to the app

### ISSUE THREAD

**Issue by mariobehling** [fossasia/badgemagic-app#638]
The proprietary app of the LED Badges has a storage functionality for 8 types. Please add this functionality to the app.

These 8 Storage places mean the the badge LED art is stored directly on the badge itself.

Compare: https://play.google.com/store/apps/details?id=com.ls.shaheen.ledcard

![Screenshot from 2020-01-30 18-16-51](https://user-images.githubusercontent.com/1583873/73473270-c5b9ec00-438c-11ea-9b3a-062a330474ef.png)

**Comment by adityastic** [comment]
@mariobehling we can create beautiful animations with this. Small 8 frame animations

**Comment by TheLastProject** [comment]
I've looked at the code a bit and right now it seems to only support setting all the messages at once.

https://github.com/fossasia/badge-magic-android/blob/0148c450a17db91b6792bfcc32fac97787f672a2/android/src/main/java/org/fossasia/badgemagic/util/SendingUtils.kt#L58-L60 takes a single message, but this should be changed to accepting a `List<Message>`.

The complexity now comes from the UI: how to make a good UI for setting up to 8 different values...

I'm willing to adopt this feature request and try to implement it myself, but it will take me some time to figure everything out.

**Comment by adityastic** [comment]
@TheLastProject you're on the right track, let me know if you need anything

**Comment by TheLastProject** [comment]
Making nice progress on this:

![photo_2020-02-03_21-08-50](https://user-images.githubusercontent.com/1885159/73687008-654ae780-46c9-11ea-91a5-ab495e2ad4e7.jpg)

TODO:
- [x] Actually being able to queue items instead of just sending all saved badges
- [ ] Showing text/clipart instead of the badge name

Hoping to be able to make a pull request adding this feature somewhere this week, but no promises obviously.

**Comment by apoorva-raj** [comment]
Hey, can I take this issue up?

**Comment by TheLastProject** [comment]
I am not a maintainer (they sadly don't seem very active), but I just want to say you're free to use as much or as little code from #644 as you see fit. I will not be offended if my word-in-progress code gets thrown away for a better implementation :)

**Comment by adityastic** [comment]
> Hey, can I take this issue up?

Yes, feel free to. Also @TheLastProject had an amazing implementation. I forgot to review your pr xD Sorry about that. It only needed a couple of additions, you can continue where he left off. I will be able to review it this weekend.

**Comment by lanrat** [comment]
Any movement on this?
